ipyvolume/pylab.py

```diff
@@ -511,26 +575,50 @@ def scatter(
     marker="diamond",
     selection=None,
     grow_limits=True,
+    lighting_model='DEFAULT',
+    opacity=1,
+    emissive_intensity=emissive_intensity_default,
+    roughness=0,
+    metalness=0,
     **kwargs
 ):
     """Plot many markers/symbols in 3d.
-
+       Due to certain shader limitations, should not use with Spot Lights and Point Lights.
+       Does not support shadow mapping.
     :param x: {x}
     :param y: {y}
     :param z: {z}
-    :param color: {color}
+    :param color: {color} Color of the material, essentially a solid color unaffected by other lighting. Default is 'red'
     :param size: {size}
     :param size_selected: like size, but for selected glyphs
     :param color_selected:  like color, but for selected glyphs
     :param marker: {marker}
     :param selection: numpy array of shape (N,) or (S, N) with indices of x,y,z arrays of the selected markers, which
                       can have a different size and color
+    :param lighting_model: The lighting model used to calculate the final color of the mesh. Can be 'DEFAULT', 'PHYSICAL'. implicit 'DEFAULT'. Will be automatically updated to 'PHYSICAL' if a light is added to figure
+    :param opacity: (Physical Only) 0 - Mesh is fully transparent; 1 - Mesh is fully opaque
+    :param emissive_intensity: (Physical Only) Factor multiplied with color. Takes values between 0 and 1. Default is 0.2
+    :param roughness: (Physical Only) How rough the material appears. 0.0 means a smooth mirror reflection, 1.0 means fully diffuse. Default is 1
+    :param metalness: (Physical Only) How much the material is like a metal. Non-metallic materials such as wood or stone use 0.0, metallic use 1.0, with nothing (usually) in between
     :param kwargs:
     :return: :any:`Scatter`
     """
     fig = gcf()
     if grow_limits:
         _grow_limits(x, y, z)
+    kwargs = kwargs.copy()
+    if lighting_model == 'DEFAULT':
+        pass  # ok, we rely on Scatter widget's default
+    elif lighting_model == 'PHYSICAL':
+        material = pythreejs.MeshPhysicalMaterial(
+                    opacity=opacity,
+                    emissiveIntensity=emissive_intensity,
+                    roughness=roughness,
+                    metalness=metalness)
+        kwargs['material'] = material
+    else:
+        # TODO: PHONG/LAMBERT
+        raise ValueError(f'Unknown lighting_model={lighting_model}')
```

Comment:
This code is quite similar to scatter, and something we want at a lot of places, maybe put it in a private function `_create_material(....)` to reuse it.